config/activity_insight.yml

```diff
@@ -1,4 +1,38 @@
 production:
+  main:
+    :username: "<%= ENV.fetch('FAMS_MAIN_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_MAIN_PASSWORD') %>"
+  webservices:
+    :username: "<%= ENV.fetch('FAMS_WEBSERVICES_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_WEBSERVICES_PASSWORD') %>"
+  backups_service:
+    :username: "<%= ENV.fetch('FAMS_BACKUPS_SERVICE_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_BACKUPS_SERVICE_PASSWORD') %>"
+  metadata_db:
+    :key: "<%= ENV.fetch('FAMS_METADATA_DB_KEY') %>"
+  s3_bucket:
+    api_key: "<%= ENV.fetch('FAMS_S3_BUCKET_API_KEY') %>"
+  lp_sftp:
+    username: "<%= ENV.fetch('FAMS_LP_SFTP_USERNAME') %>"
+    host: "<%= ENV.fetch('FAMS_LP_SFTP_HOST') %>"
+development:
+  main:
+    :username: "<%= ENV.fetch('FAMS_MAIN_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_MAIN_PASSWORD') %>"
+  webservices:
+    :username: "<%= ENV.fetch('FAMS_WEBSERVICES_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_WEBSERVICES_PASSWORD') %>"
+  backups_service:
+    :username: "<%= ENV.fetch('FAMS_BACKUPS_SERVICE_USERNAME') %>"
+    :password: "<%= ENV.fetch('FAMS_BACKUPS_SERVICE_PASSWORD') %>"
+  metadata_db:
+    :key: "<%= ENV.fetch('FAMS_METADATA_DB_KEY') %>"
+  s3_bucket:
+    api_key: "<%= ENV.fetch('FAMS_S3_BUCKET_API_KEY') %>"
+  lp_sftp:
+    username: "<%= ENV.fetch('FAMS_LP_SFTP_USERNAME') %>"
+    host: "<%= ENV.fetch('FAMS_LP_SFTP_HOST') %>"
+test:
   main:
```

Comment:
We should probably change all the test values to some fake placeholder like "test".  Just to be sure we don't accidentally have a test send live requests — even though our external requests should be stopped and mocked by WebMock.